mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -237,8 +237,10 @@
   private List<JobExecution> prepareJobExecutionList(String parentJobExecutionId, List<File> files, UserInfo userInfo, InitJobExecutionsRqDto dto) {
     String userId = dto.getUserId();
     if (dto.getSourceType().equals(InitJobExecutionsRqDto.SourceType.ONLINE)) {
+      JobProfileInfo jobProfileInfo = dto.getJobProfileInfo();
+      jobProfileInfo.withName("CLI job profile");
```

Comment:
Discuss the name with AM

Reply:
@KaterynaSenchenko Done. I changed this name.